### DISCUSSION TITLE
How to get response from github support ?

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#124579] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've had a ticket open since February 13, but I haven't received any response from the support team yet. I pay for a PRO GitHub account specifically to receive their standard support, but it's been 3 days, and they still haven't responded. I'm tired of asking in the community about my issue, and their community employees keep giving the same answer and closing my discussions.

**Comment by asmawi117** [comment]
My account was hacked and subsequently suspended, with the hacker changing both my email address and password. Some of the GitHub community staff responded to my earlier discussions and advised me to use their report abuse tool. I have also tried to appeal for reinstatement, but so far, there has been no progress. Do they have the ability to flag this to their support team?

It wasn't just me who got hacked; two of my friends (classmates) were also hacked. One of my friend's cases was resolved in March, but mine and my other friend's cases still haven't been solved yet.

**Comment by davevad93** [comment]
Hi @asmawi117 ,

That is unfortunate, maybe it sounds obvious but have you tried contacting them directly by email? A friend of mine once did that and they replied within a couple of days, it's possible that they did it so quickly because he's an Accenture employee, but give it a try anyway and see if you have any luck. 

[support@github.com](mailto:support@github.com)

**Reply by asmawi117** [reply]
I contacted them by email 2 months ago. I also contacted them using my university email, which was connected to my hacked account.

**Reply by davevad93** [reply]
Ah ok, I'm sorry for that @asmawi117 I can feel your frustration and I hope you can solve this as quickly as possible. In any case I would send another email, and if in a while they still don't reply I would send another one, they probably get a lot of emails every day and it's likely they will never see the one you sent them 2 months ago.

**Comment by asmawi117** [comment, marked as answer]
GitHub account has finally been recovered by GitHub support after a waiting period of 4 months and 4 days.

**Comment by arcane234** [comment]
@davevad93  its been more than 40 days and still haven't recovered my dublicate charges, I'm really disheartened https://support.github.com/ticket/personal/0/4091973
